js/tests/e2e/global-setup.js

```diff
@@ -34,35 +34,42 @@ const isTestAppAvailable = async (appName) => {
   return applications.find((application) => application.name === appName);
 };
 
-const hasTestSession = async (appName) => {
-  const response = await fetch(`${BASE_URL}/sessions`);
-  const sessions = await response.json();
-  return sessions.find((session) => session.app === appName);
+const hasTestInstance = async (appName) => {
+  const response = await fetch(`${BASE_URL}/instances`);
+  const instances = await response.json();
+  return instances.find((instance) => instance.app_name === appName);
 };
 
-const hasActiveTestSession = async (appName) => {
-  const response = await fetch(`${BASE_URL}/sessions`);
-  const sessions = await response.json();
-  const session = sessions.find(
-    (session) => session.app === appName && session.status === "active",
+const getRunningSessionId = async (appName) => {
```

Comment:
```suggestion
const getRunningInstanceId = async (appName) => {
```
Should this name be aligned as well?

Reply:
No, this method returns a session id, not an instance id.